man/systemd-sysext.xml

```diff
@@ -205,6 +212,51 @@
     to tie the most frequently configured options to runtime updateable flags that can be changed without a
     system reboot. This will help reduce servicing times when there is a need for changing the OS configuration.</para></refsect1>
 
+  <refsect1>
+    <title>Mutability</title>
+    <para>By default, merging system extensions on mutable host file systems will render <filename>/usr/</filename>
+    and <filename>/opt/</filename> hierarchies read-only. Merging configuration extensions will have the same
+    effect on <filename>/etc/</filename>. Mutable mode allows writes to these locations when extensions are
+    merged.</para>
+
+    <para>The following modes are supported:
+    <orderedlist>
+      <listitem><para><option>auto</option>: Automatic mode; the default. Mutability is disabled by default
+        and only enabled if a corresponding write routing directory exists below
+        <filename>/var/lib/extensions.mutable/</filename>.</para></listitem>
+      <listitem><para><option>enabled</option>: Force mutable mode and automatically create write routing
+        directories below <filename>/var/lib/extensions.mutable/</filename> when required.</para></listitem>
+      <listitem><para><option>disabled</option>: Force immutable mode even if write routing
+        directories exist below <filename>/var/lib/extensions.mutable/</filename>.</para></listitem>
+      <listitem><para><option>import</option>: Force immutable mode like above, but merge the contents of
+        directories below <filename>/var/lib/extensions.mutable/</filename> into the host file system.</para></listitem>
+    </orderedlist>
+    See "Options" below on specifying modes using the <option>--mutable=</option> command line option.</para>
+
+    <para>Mutable mode routes writes to subdirectories in <filename>/var/lib/extensions.mutable/</filename>.
+    <simplelist type="horiz">
+      <member>Writes to <filename>/usr/</filename> are directed to <filename>/var/lib/extensions.mutable/usr/</filename></member>,
+      <member>writes to <filename>/opt/</filename> are directed to <filename>/var/lib/extensions.mutable/opt/</filename>, and</member>
+      <member>writes to <filename>/etc/</filename> land in <filename>/var/lib/extensions.mutable/etc/</filename>.</member>
+    </simplelist></para>
+
+    <para>If <filename>usr/</filename>, <filename>opt/</filename>, or <filename>etc/</filename>
+    in <filename>/var/lib/extensions.mutable/</filename> are symlinks, then writes are directed to the
+    symlinks' targets.
+    Consequently, to retain mutability of a host file system, create symlinks
+    <simplelist type="horiz">
+      <member><filename>/var/lib/extensions.mutable/etc/</filename> -> <filename>/etc/</filename></member>
+      <member><filename>/var/lib/extensions.mutable/usr/</filename> -> <filename>/usr/</filename></member>
+      <member><filename>/var/lib/extensions.mutable/opt/</filename> -> <filename>/opt/</filename></member>
```

Comment:
ahum. this is not acceptable. In this codebase we love Unicode. Hence please use proper unicode arrows, here have three of them to copy in: → → →